app/dashboard/views.py

```diff
@@ -2776,3 +2785,32 @@ def get_hackathons(request):
         'hackathons': events,
     }
     return TemplateResponse(request, 'dashboard/hackathons.html', params)
+
+@require_POST
+@login_required
+def upload_profile_banner(request):
```

Comment:
So given that there is no longer the ability to upload a custom image, do we still need this view and endpoint?

Reply:
I'll remove it now.

Reply:
This endpoint updates the profile of the user.